BusyList/Handlers/EditHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BusyList.Commands;
+using System;
+using System.Reflection;
+
+namespace BusyList.Handlers
+{
+    // There are some problems. 
+    // Only work for string properties e.g description.
+    public class EditHandler : IHandler<EditCommand>
+    {
+        private readonly ITaskRepository _taskRepository;
+
+        public EditHandler(ITaskRepository taskRepository)
+        {
+            _taskRepository = taskRepository;
+        }
+
+        public void Run(EditCommand command)
+        {
+            TaskItem taskItem = _taskRepository.GetTaskById(command.Id);
+
+            if (taskItem != null)
```

Comment:
This is just a style suggestion, but, you could invert the if statements, that way the else statments can be removed and nesting can be reduced  :)
```suggestion
           if (taskItem == null)
            {
                Console.WriteLine($"The task with the id {command.Id} does not exist");
                return;
            }

            PropertyInfo propertyInfo = taskItem.GetType().GetProperty(command.property);

            if (propertyInfo == null)
            {
                Console.WriteLine($"The property with the name {command.property} does not exist");
                return;
            }
            
            propertyInfo.SetValue(taskItem, command.value);
            _taskRepository.UpdateTask(taskItem);
            Console.WriteLine("Task updated");
```